feat(router): enabling MinimalRouterStateSerializer by default (#2326)
ngrx/platform: projects/ngrx.io/content/guide/router-store/configuration.md

## Default Router State Serializer

`DefaultRouterStateSerializer` router state serializer, serializes the URL together with the [ActivatedRouteSnapshot](https://angular.io/api/router/ActivatedRouteSnapshot) from [Angular Router](https://angular.io/guide/router). The latter is serialized recursively, but only with the possibility to traverse the route downward since `root` and `parent` parameters are set to `undefined`.

<div class="alert is-important">

The `DefaultRouterStateSerializer` cannot be used when [serializability runtime checks](guide/store/configuration/runtime-checks) are enabled.

> With runtime checks enabled `MinimalRouterStateSerializer` serializer is used by default in `RouterStoreModule` when no other serializer is provided. This also applies to Ivy with immutability runtime checks.

Reviewer: ```suggestion
With serializability runtime checks enabled, the `MinimalRouterStateSerializer` serializer **must** be used. This also applies to Ivy with immutability runtime checks.
```

import { RouterStateSerializer } from '@ngrx/router-store';

export interface RouterStateUrl {

> url: string;

Reviewer: Seems like something weird has happened here with the formation, could you revert this change please?

params: Params;
queryParams: Params;
}

export class CustomSerializer implements RouterStateSerializer&lt;RouterStateUrl&gt; {

> serialize(routerState: RouterStateSnapshot): RouterStateUrl {

Reviewer: Same here

    // Only return an object including the URL, params and query params
    // instead of the entire snapshot
    return { url, params, queryParams };

> }

Reviewer: And here
